### ISSUE TITLE
Migrator.net does not work with this any more

### ISSUE THREAD

**Issue by jogibear9988** [aspnet/Microsoft.Data.Sqlite#529]
Relate dto : https://github.com/aspnet/Microsoft.Data.Sqlite/issues/484

**Comment by mdisg** [comment]
A possible solution could be a constructor overload which controls the disposal of the created data reader if someone needs it.

```
public SqliteCommand(bool cleanUpAllResourcesOnDispose)
{
    ...
}
```

**Comment by jogibear9988** [comment]
Yes. But the default should be the old behavior

**Comment by jogibear9988** [comment]
@ajcvickers how can we solve this?

**Comment by mdisg** [comment]
@jogibear9988 yes, i agree with that would also be good to here the thoughts of the developer who changed the behavior @bricelam

**Comment by jogibear9988** [comment]
I think the new behavior will break many existing projects.
